config/checkstyle/checkstyle.xml

```diff
@@ -190,17 +190,20 @@
                value="Local variable name ''{0}'' must match pattern ''{1}''."/>
     </module>
     <module name="ClassTypeParameterName">
-      <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>
+      <property name="format"
+        value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)|([A-Z][a-zA-Z0-9]*Type$)"/>
```

Comment:
Allow generic types of the form "InputType"

Reply:
Question> Do we move this into our code style repository?

Reply:
probably we should